plugins/Resolution/Columns/Resolution.php

```diff
@@ -39,4 +39,20 @@ public function onNewVisit(Request $request, Visitor $visitor, $action)
 
         return $resolution;
     }
+    
+    /**
+     * @param Request $request
+     * @param Visitor $visitor
+     * @param Action|null $action
+     * @return mixed 
+     */
+    public function onExistingVisit(Request $request, Visitor $visitor, $action)
+    {
+        // In case the value was initially unknown, update it from a subsequent page impression
```

Comment:
It would actually be updated with any following action, if the parameter is provided.
```suggestion
        // In case the value was initially unknown, update it from a subsequent action
```